ckanext/datagovuk/templates/user/new_user_form.html

```diff
@@ -0,0 +1,22 @@
+{% import "macros/form.html" as form %}
+
+<form id="user-register-form" class="form-horizontal" action="" method="post">
+  {{ form.errors(error_summary) }}
+  {{ form.input("name", id="field-username", label=_("Username"), placeholder=_("username"), value=data.name, error=errors.name, classes=["control-medium"]) }}
+  {{ form.input("fullname", id="field-fullname", label=_("Full Name"), placeholder=_("Joe Bloggs"), value=data.fullname, error=errors.fullname, classes=["control-medium"]) }}
+  {{ form.input("email", id="field-email", label=_("Email"), type="email", placeholder=_("joe@example.com"), value=data.email, error=errors.email, classes=["control-medium"]) }}
+  {{ form.input("password1", id="field-password", label=_("Password"), type="password", placeholder="••••••••", value=data.password1, error=errors.password1, classes=["control-medium"], attrs={'pattern': '.{8,}', 'title': 'Password must be a minimum of 8 characters'}) }}
+  {{ form.input("password2", id="field-confirm-password", label=_("Confirm"), type="password", placeholder="••••••••", value=data.password2, error=errors.password1, classes=["control-medium"], attrs={'pattern': '.{8,}', 'title': 'Password must be a minimum of 8 characters'}) }}
```

Comment:
`error=errors.password2`?

Reply:
You're absolutely right.  This was copied straight out of the core CKAN code to override in our plugin, so it must be incorrect there too.

Reply:
Nice open-source opportunity for you there then!